internal/scorecard/plugins/plugin_runner.go

```diff
@@ -295,14 +299,18 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 			if schelpers.IsV1alpha2(config.Version) {
 				basicTests.ApplySelector(config.Selector)
 			}
-			basicTests.Run(context.TODO())
-			logs, err := ioutil.ReadAll(logReadWriter)
-			if err != nil {
-				basicTests.Log = fmt.Sprintf("failed to read log buffer: %v", err)
+			if config.ListOpt {
+				basicTests.List()
```

Comment:
in this version the text output with the --list flag looks like this:

basic:
	Spec Block Exists                   
	Labels: 
		necessity:required
		suite:basic

	Status Block Exists                 
	Labels: 
		necessity:required
		suite:basic

	Writing into CRs has an effect      
	Labels: 
		necessity:required
		suite:basic

olm:
	Provided APIs have validation       
	Labels: 
		necessity:required
		suite:olm

	Owned CRDs have resources listed    
	Labels: 
		necessity:required
		suite:olm

	CRs have at least 1 example         
	Labels: 
		necessity:required
		suite:olm

	Spec fields with descriptors        
	Labels: 
		necessity:required
		suite:olm

	Status fields with descriptors      
	Labels: 
		necessity:required
		suite:olm

Reply:
this is the output when --list is not specified:

basic:
	Spec Block Exists                   : pass
	Labels: 
		necessity:required
		suite:basic

	Status Block Exists                 : pass
	Labels: 
		necessity:required
		suite:basic

	Writing into CRs has an effect      : pass
	Labels: 
		suite:basic
		necessity:required

olm:
	Spec fields with descriptors        : pass
	Labels: 
		necessity:required
		suite:olm

	Status fields with descriptors      : fail
	Labels: 
		suite:olm
		necessity:required
	Suggestions: 
		Add a status descriptor for status

	Provided APIs have validation       : pass
	Labels: 
		necessity:required
		suite:olm

	Owned CRDs have resources listed    : pass
	Labels: 
		necessity:required
		suite:olm
	Suggestions: 
		If it would be helpful to an end-user to understand or troubleshoot your CR, consider adding resources [configmaps/v1 memcacheds/v1alpha1 replicasets/v1 deployments/v1 services/v1] to the resources section for owned CRD Memcached

	CRs have at least 1 example         : pass
	Labels: 
		necessity:required
		suite:olm